python/pyspark/ml/stat.py

```diff
@@ -71,6 +71,62 @@ def test(dataset, featuresCol, labelCol):
         return _java2py(sc, javaTestObj.test(*args))
 
 
+class Correlation(object):
+    """
+    .. note:: Experimental
+
+    Compute the correlation matrix for the input dataset of Vectors using the specified method.
+    Methods currently supported: `pearson` (default), `spearman`.
```

Comment:
So the Scala documentation had a warning about caching being suggested when using Spearman, would it make sense to copy this warning over as well?